unittest/python/test_contacts_2D.py

```diff
@@ -0,0 +1,92 @@
+import numpy.core.multiarray
+import os
+import sys
+
+import numpy as np
+
+import pinocchio
+from pinocchio.utils import rand,zero
+import crocoddyl
+import matplotlib.pylab as plt
+from numpy.linalg import norm, solve
+
+sys.path.append("/local/src/crocoddyl/unittest/")
+from testutils import NUMDIFF_MODIFIER, assertNumDiff, df_dq, df_dx
+pinocchio.switchToNumpyMatrix()
+
+def m2a(m): return np.array(m.flat)
+
+def a2m(a): return np.matrix(a).T
+
+EPS = np.finfo(float).eps
+NUMDIFF_MODIFIER = 1e4
+
+# Loading Talos arm with FF TODO use a bided or quadruped
+# -----------------------------------------------------------------------------
+# Load collision model
+URDF_FILENAME = "solo.urdf"
+SRDF_FILENAME = "solo.srdf"
+SRDF_SUBPATH = "/solo_description/srdf/" + SRDF_FILENAME
+URDF_SUBPATH = "/solo_description/robots/" + URDF_FILENAME
+
+modelPath = "/opt/openrobots/share/example-robot-data/robots"
```

Comment:
You should use `import example_robot_data`  and do a `loadSolo` to load the model. These custom paths may be different based on user.